script/test

```diff
@@ -7,3 +7,5 @@ if [ -n "$SAUCE_BROWSER" ]; then
 else
   ./script/phantomjs
 fi
+
+./node_modules/mocha/bin/mocha test/cjs.js
```

Comment:
Why is this separate file needed?

These tests could be in `test.js`. That file [already jumps through hoops](https://github.com/github/fetch/blob/cbb313bf09512efda9444e40fe9e8ac432ac1108/test/test.js#L83-L95) to exercise both native and polyfilled fetch, and this module approach could potentially clean it up.

Reply:
A separate file was used as it is testing whether the CJS exports are available and the easiest and foolproof way seemed to be to do it inside a real CJS environment. test.js is evaluated in the context of a browser, where no `require` exists, and I did not see any other tests running inside a CJS env. Sorry about that, am I missing an easier solution?

Reply:
@mislav any thoughts? 🕺

Reply:
bump